crmsh/bootstrap.py

```diff
@@ -2138,11 +2163,22 @@ def bootstrap_join(cluster_node=None, ui_context=None, nic=None, quiet=False, ye
         join_ssh(cluster_node)
         join_csync2(cluster_node)
         join_ssh_merge(cluster_node)
+        join_remote_auth(cluster_node)
         join_cluster(cluster_node)
 
     status("Done (log saved to %s)" % (LOG_FILE))
 
 
+def join_remote_auth(node):
+    invoke("rm -f {}".format(PCMK_REMOTE_AUTH))
+    pcmk_remote_dir = os.path.dirname(PCMK_REMOTE_AUTH)
+    if not os.path.exists(pcmk_remote_dir):
+        invoke("mkdir -p --mode=0750 {}".format(pcmk_remote_dir))
+        invoke("chgrp haclient {}".format(pcmk_remote_dir))
+
+    invoke('ssh root@{remote} "csync2 -mr {file}; csync2 -fr {file}; csync2 -rxv {file}"'.format(remote=node, file=PCMK_REMOTE_AUTH))
```

Comment:
This seems convoluted and unnecessary, the authkey should be included in the basic HA csync2 group so that it is synced automatically.

Reply:
No, without this last line, /etc/pacemaker/authkey wouldn't sync to the join-node.

Reply:
There is another problem if that is the case, I don't think this is the right solution.